UnityProject/Assets/Scripts/Core/GameGizmos/GameGizmoModel.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameGizmoModel
+{
+	public string Ver = "1.0.0";
+	public string Pos;
```

Comment:
ew, just create two variables, one that is actually the vector3 type and the other that is the string that reads from the vec3 for saving

Reply:
that juust makes saving more fiddly tho